content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx

```diff
@@ -423,54 +425,20 @@ model Category {
 }
 ```
 
-### Relation tables
-
-A relation table (also sometimes called _JOIN_, _link_ or _pivot_ table) connects two or more other tables and therefore creates a _relation_ between them. Creating relation tables is a common data modeling practice in SQL to represent relationships between different entities. In essence it means that "one m-n relation is modeled as two 1-n relations in the database".
-
-When using Prisma, you can create relation tables by defining [models](/concepts/components/prisma-schema/data-model#defining-models) similar to how you would model them as tables. The main difference is that the fields of the relation table are both annotated relation fields with a corresponding relation scalar field.
-
-Relation tables are also often used to add "meta-information" to a relation. For example, to store _when_ the relation was created.
-
-Here is an example for a relation table called `CategoriesOnPosts`:
-
-```prisma
-model Post {
-  id         Int                 @id @default(autoincrement())
-  title      String
-  categories CategoriesOnPosts[]
-}
+## Relation tables
 
-model Category {
-  id    Int                 @id @default(autoincrement())
-  name  String
-  posts CategoriesOnPosts[]
-}
-
-model CategoriesOnPosts {
-  post       Post     @relation(fields: [postId], references: [id])
-  postId     Int
-  category   Category @relation(fields: [categoryId], references: [id])
-  categoryId Int
-  assignedAt DateTime @default(now())
-
-  @@id([postId, categoryId])
-}
-```
-
-In this example, the `assignedAt` field adds additional information about the relation between `Post` and `Category`: it specifies when the post was assigned to the category.
-
-Note that the same rules as for [1-n-relations](one-to-many-relations) apply (because `Post`↔ `CategoriesOnPosts` and `Category` ↔ `CategoriesOnPosts` are both in fact 1-n-relations), which means one side of the relation needs to be annotated with the `@relation` attribute.
+A relation table (also sometimes called _JOIN_, _link_ or _pivot_ table) connects two or more other tables and therefore creates a _relation_ between them. Creating relation tables is a common data modelling practice in SQL to represent relationships between different entities. In essence it means that "one m-n relation is modeled as two 1-n relations in the database".
```

Comment:
```suggestion
A relation table (also sometimes called a _JOIN_, _link_ or _pivot_ table) connects two or more other tables and therefore creates a _relation_ between them. Creating relation tables is a common data modelling practice in SQL to represent relationships between different entities. In essence it means that "one m-n relation is modeled as two 1-n relations in the database".
```

Reply:
Is there something that I'm missing? This is grammatically incorrect without either the 'a' or modifying 'table' to be plural